Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyViewContextMenuPlugin.cxx

```diff
@@ -256,8 +292,32 @@ void qSlicerSubjectHierarchyViewContextMenuPlugin::showViewContextMenuActionsFor
 
   d->CopyImageAction->setVisible(true);
 
+  // Check tilt lock in camera widget and set menu item accordingly
   bool isSliceViewNode = (vtkMRMLSliceNode::SafeDownCast(viewNode) != nullptr);
   d->ConfigureSliceViewAnnotationsAction->setVisible(isSliceViewNode);
+  d->FitSliceViewAction->setVisible(isSliceViewNode);
+  d->CenterThreeDViewAction->setVisible(!isSliceViewNode);
+
+  d->ToggleTiltLockAction->setVisible(!isSliceViewNode);
+  if (!qSlicerApplication::application()
+    || !qSlicerApplication::application()->layoutManager())
+  {
```

Comment:
Follow the brace indentation style of the existing code base. We'll modernize this at some point (will change it to the bracing style that you used in this commit), but until then we keep the indentation as is.